Dockerfile

```diff
@@ -10,25 +28,17 @@ RUN set -eux; \
 		ca-certificates \
 		wget \
 		\
-		# bzr \
-		# git \
-		# mercurial \
-		openssh-client \
-		# subversion \
-		\
-# we need "gsfonts" for generating PNGs of Gantt charts
-# and "ghostscript" for creating PDF thumbnails (in 4.1+)
+    # we need "gsfonts" for generating PNGs of Gantt charts
+    # and "ghostscript" for creating PDF thumbnails (in 4.1+)
 		ghostscript \
 		gsfonts \
 		imagemagick \
-# https://github.com/docker-library/ruby/issues/344
-		shared-mime-info \
-# grab gosu for easy step-down from root
+    # grab gosu for easy step-down from root
 		gosu \
-# grab tini for signal processing and zombie killing
+    # grab tini for signal processing and zombie killing
 		tini \
 	; \
-# allow imagemagick to use ghostscript for PDF -> PNG thumbnail conversion (4.1+)
+  # allow imagemagick to use ghostscript for PDF -> PNG thumbnail conversion (4.1+)
```

Comment:
@dkastl This is not the big issue, but it may be good to start comment `#` without indent.